src/Icons/src/IconRenderer.php

```diff
@@ -54,10 +48,8 @@ private function getIconAttributes(string $name, array $attributes): array
         $iconAttributes = $this->defaultIconAttributes;
 
         // Add aria-hidden attribute
-        if ($this->rendererConfiguration['auto_aria_hidden'] ?? false) {
-            if ([] === array_intersect(['aria-hidden',  'aria-label', 'aria-labelledby', 'title'], array_keys($attributes))) {
-                $iconAttributes['aria-hidden'] = 'true';
-            }
+        if ([] === array_intersect(['aria-hidden',  'aria-label', 'aria-labelledby', 'title'], array_keys($attributes))) {
```

Comment:
simple curiosity where did you get this list?

Reply:
> An image’s alt attribute.
    A SVG’s child [title element](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title).
    Using aria-label or aria-labelledby on an image or SVG.
    Using [a title attribute](https://w3c.github.io/html-aam/#img-element) on an image or SVG.
    
https://www.scottohara.me/blog/2019/05/22/contextual-images-svgs-and-a11y.html